### ISSUE TITLE
popsCwcheat is broken

### ISSUE THREAD

**Issue by GoogleCodeExporter** [kuuhakuzx/procfw#539]
```
What steps will reproduce the problem?
1. Make an eboot of any PSX game or use any PSN dowload
2. boot it and press & holde select
3. go to select cheats you will see a huge blank cheat list.

What is the expected output? What do you see instead?
I am expecting to see cheats for any of my 3 PSX games however they dont show 
up despite i have them in my popscwcheat correctly labelled.

What version of the product are you using? On what operating system?

CWcheat 2.3, PRO-C2 CFW

Please provide any additional information below.

I was able to fix it somehow once before but i accidentally re ran the upgrade 
app of the PRO-C2 CFW so it's pretty much broken.

In short the CFW isnt interacting with the popscwcheat file correctly, oddly it 
is interacting correctly with the Cwcheat for PSP games.
```

Original issue reported on code.google.com by `lxfoss...@gmail.com` on 21 Feb 2013 at 2:38

**Comment by GoogleCodeExporter** [comment]
```
This is not a CFW issue, but a plugin issue. You should post it where the 
plugin is developed, not here.
```

Original comment by `piotrekhenry` on 22 Feb 2013 at 11:20

**Comment by GoogleCodeExporter** [comment]
```
Test it at PRO-C1, if the bug is still there, its a plugin issue.
If the bug is only at C2, its a bug due to the changes between C1 and C2.
```

Original comment by `mirzet.w...@gmx.de` on 23 Feb 2013 at 8:20

**Comment by GoogleCodeExporter** [comment]
```
I went from pro C-1 to pro C-2 and the issue repeats itself, only on pro c2, i 
found a temporary fix, i think the CFW isnt doing a procedure correctly, the 
fix is merley run the update a second time ontop of the first, this makes the 
popscwcheat to be read properly, i tested a number of things before posting the 
issue here:

i swapped cwcheatpops files, i re organized the popsfile by alphabetical order, 
followed by that i did a ISO mode test and then i removed and re added the 
game, none of it worked until i re did the install, i even went and disabled my 
only plugins (POPS and CWcheat i dont run any others), as i said it was the 
second reinstall that made the whole thing work.
```

Original comment by `lxfoss...@gmail.com` on 25 Feb 2013 at 6:09

**Comment by GoogleCodeExporter** [comment]
```
Reporting that the bug popped up again on PRO C-2
```

Original comment by `lxfoss...@gmail.com` on 20 Jul 2013 at 4:41
